src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+html {
+  font-family: Roboto, Arial, sans-serif;
+  --color-blue: #00acdc;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo__icon {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item {
+  display: flex;
+  position: relative;
+  margin-right: 20px;
+}
```

Comment:
![image](https://user-images.githubusercontent.com/97547824/201185976-75e36397-7ba0-433a-872b-b43c5b6357e6.png)
Remove margin for last item